index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>title</title>
+        <link rel="stylesheet" href="styles.css">
+    </head>
+<body>
+    <header id="header">
+      <img id="header-img" src="https://d92mrp7hetgfk.cloudfront.net/images/sites/misc/SmallerMicroverselogo_revised/original.jpg?1581442029" alt="Logo Microverse" />
+      <nav id="nav-bar">
+        <ul>
+          <li class="nav-link">Features</li>
+          <li class="nav-link">How it works</li>
+          <li class="nav-link">Pricing</li>
+        </ul>
+      </nav>
+      <iframe width="560" height="315" src="https://www.youtube.com/embed/y8Yv4pnO7qc" title="Roman Carnival Overture Op. 9 for Five Trombones" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+      <form id="form" action="https://www.freecodecamp.com/email-submit" method="POST">
+        <label>Email <input id="email" placeholder="Enter your email" type="email"></input>
+        </label>
+        <label> <input id="submit" type="submit"></input>
+        </label>
+      </form>
+    </header>
```

Comment:
I think the semantic tags could be implemented  differently. It will help to call different parts of the site and not only the header.

Reply:
- [ ] @jlberbesi Great job explaining why you requested the changes 👏 but kindly make sure your comments are straightforward and easy to understand. For Example:

> * **_This is a clear comment:_** *The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?*
> * **_This is a vague comment:_** *The README is not descriptive, why don't you make it better?*

Reply:
Comment was updated from initial version of it.